lib/component.js

```diff
@@ -505,10 +507,34 @@ class Component extends WebComponent {
         attrValue = this.getJSONAttribute(attr);
       }
 
-      this.attrs[attr] = attrValue;
+      this._attrs[attr] = attrValue;
+
+      if (attrSchema.deprecatedMsg) {
+        console.warn(`attr '${attr}' is deprecated. ${attrSchema.deprecatedMsg}`);
```

Comment:
great idea

Reply:
Would probably be useful to include the component name in the message